src/modes/single/SingleLocation.svelte

```diff
@@ -11,20 +11,29 @@
   } from '../../stores';
   import { sensorList } from '../../stores/constants';
   import SensorCard from './SensorCard.svelte';
-  import throttle from 'lodash-es/throttle';
+  import debounce from 'lodash-es/throttle';
   import { selectionColors } from '../../theme';
   import { resolveHighlightedTimeValue } from '../overview/vegaSpec';
 
   let highlightTimeValue = null;
 
-  const throttled = throttle((value) => {
-    highlightTimeValue = value;
-  }, 25);
+  // Each mouse move across the chart results in a mouseout for previous highlight
+  // date immediately followed by mouseover for the next date, if any.  We need to
+  // ignore the first event unless there is no second.  A very short debounce time
+  // achieves that goal, at least most of the time.
+  const debouncedHighlightTime = debounce(
```

Comment:
why is a debouncing better than a throttling?

Reply:
Throttling will reduce the rate that it lets events through, so it might pick the first (clear) event or the second event (for the mouse over a new date).  We only want the second one of each pair, if there is a second one, otherwise we want the first one.  Debouncing throws away extra events that arrive too quickly, which are the 'bounces' when you really do have bouncing events.  In this situation, the two events are very close, so we can have the debouncing always throw away the first event.